react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,19 +1,24 @@
+/* eslint-disable react/jsx-no-bind */
+/* eslint-disable react/prop-types */
 import React, { Component } from 'react';
 
 import Square from '../Square';
 
 import styles from './styles.module.scss';
 
 class Board extends Component {
-  renderSquare() {
-    return <Square />;
+  renderSquare(i) {
+    return (
+      <Square
+        value={this.props.squares[i]}
+        onClick={() => this.props.onClick(i)}
```

Comment:
Creating arrow functions in renders generates a creation of them with each render.
How do you think we can solve this problem?

Reply:
I undertand u. I'm gonna put that logic in a `handleClick` method